src/main/java/mate.academy/service/impl/ProductParserImpl.java

```diff
@@ -5,7 +5,6 @@
 import mate.academy.service.ProductParser;
 
 public class ProductParserImpl implements ProductParser {
```

Comment:
Do we need annotation here? think about it

Reply:
What about FileReaderService?